boards/stm32l476g-disco/include/board.h

```diff
@@ -7,7 +7,7 @@
  */
 
 /**
- * @defgroup    boards_stm32l476g-disco STM32L476G-DISCO
+ * @ingroup    boards_stm32l476g-disco STM32L476G-DISCO
```

Comment:
```suggestion
 * @ingroup    boards_stm32l476g-disco
```

Reply:
Ok. Maybe this resolve the problem with static tests ;) - but now doc page looks good.